content/TCGettingStarted/TCReleaseNotes.md

```diff
@@ -1,9 +1,131 @@
+
+Starting in version 3.1.0, TrueCommand supports STARTTLS again to support availability for Microsoft365 users. 
+
+Users can now enable or disable snapshot tasks directly, providing better control over scheduled backups.
+
+Enhancements to CPU reporting and Disk IOPs reports provide more accurate system performance data.
+
+A new NAS OTP code dialog improves two-factor authentication workflows and temporary password resets for Cloud users are fixed.
+
+Note that if you are using TrueCommand with a TrueNAS SCALE HA system, you need to update SCALE to version 23.10 or later before connecting to TrueCommand 3.1.0.
+
+API key logins now fail and revoke the key, triggering a NAS alert. Users who have swtiched from the default SSL port, or are using a self-signed certificate, might experience broken connections. It is advised to update your authentication settings prior to updating to TrueCommand 3.1.0 to avoid this disruption.
+
+After updating, clear the browser cache (CTRL+F5) before logging in to TrueCommand. This ensures stale data does not interfere the TrueCommand UI.
+
+### TrueNAS Compatibility
+
+TrueCommand 3.1.0 is tested and compatible with these TrueNAS versions:
+
+* 13.0
+* 22.12
+* 23.10
+* 24.04
+* 24.10
+* 25.04
+
+### 3.1.0 Paths
+
+Self-hosted Containers:
+```mermaid
+flowchart LR
+A["Legacy (Pre 1.2)"] --> B
+B["1.3"] --> C
+C["2.3.3"] --> D
+D["3.0.2"] --> E
+E["3.1.0"]
+```
+
+## Release Schedule
+
+{{< include file="/static/includes/ReleaseScheduleWarning.md" >}}
+
+{{< releaselist name=tc-releases defaultTab=2 >}}
+
+{{< expand "Software Lifecycle" "v" >}}
+{{< include file="/static/includes/LifecycleTable.md" >}}
+{{< include file="/static/includes/SoftwareStatusPage.md" >}}
+{{< /expand >}}
+
+## 3.1.0 Changelog
+
+**March 4, 2025**
+
+iXsystems is pleased to release TrueCommand 3.1.0!
+
+Notable changes:
+
+* Anticipate insecure logins on FT+ ([TC-3273](https://ixsystems.atlassian.net/browse/TC-3273))
+* Update usage of HostInfo product field ([TC-3272](https://ixsystems.atlassian.net/browse/TC-3272))
+* Add dialog to provide NAS OTP code ([TC-3268](https://ixsystems.atlassian.net/browse/TC-3268))
+* Create Disk IOPs Report ([TC-3255](https://ixsystems.atlassian.net/browse/TC-3255))
+* UI hangs trying to display large middleware log ([TC-3264](https://ixsystems.atlassian.net/browse/TC-3264))
+* Fangtooth Compatibility ([TC-3263](https://ixsystems.atlassian.net/browse/TC-3263))
+* Add cloud-specific wording to system delete ([TC-3242](https://ixsystems.atlassian.net/browse/TC-3242))
+* Improve logging for system events ([TC-3281](https://ixsystems.atlassian.net/browse/TC-3281))
+* Resolve middleware crash on large dataset replication ([TC-3276](https://ixsystems.atlassian.net/browse/TC-3276))
+* Enhance security checks for API requests ([TC-3275](https://ixsystems.atlassian.net/browse/TC-3275))
+* Fix UI inconsistencies in dashboard graphs ([TC-3274](https://ixsystems.atlassian.net/browse/TC-3274))
```

Comment:
Again this is just part of FT compatibility, relating to the reporting.realtime output change